### PR TITLE
refactor(dynamic_scale_down_mgr): quote the cluster_id to deal with the case of an empty string

### PR DESCRIPTION
**Pull request by machi1990**
…

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] All PR comments are resolved either by addressing them or creating follow up tasks
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has been created for changes required on the client side

### CODE REVIEW

**Review by pawelpaszki — APPROVED** [review]

File: internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go [hunk 1]
```diff
@@ -302,14 +302,14 @@ func (p *standardDynamicScaleDownProcessor) createNewStreamingUnitPerClusterList
...
-			glog.V(10).Infof("ignoring cluster with cluster_id %s in terraforming state %s:", suCount.ClusterId, suCount.Status)
+			glog.V(10).Infof("ignoring cluster with cluster id '%s' in terraforming state %s:", suCount.ClusterId, suCount.Status)
 			continue
 		}
 
 		// Keep only clusters streaming unit count for cluster that are not going to be deleted.
 		// i.e Assume that the candidate cluster is removed from the new list which will be used to perform scale up evaluation
 		if suCount.ClusterId == p.clusterID {
-			glog.V(10).Infof("skipping candidate cluster with cluster_id %s:", suCount.ClusterId)
+			glog.V(10).Infof("skipping candidate cluster with cluster id '%s':", suCount.ClusterId)
```
Comment (JameelB) [inline internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go:312]:
```suggestion
			glog.V(10).Infof("skipping candidate cluster with cluster id '%s'", suCount.ClusterId)
```

File: internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go [hunk 2]
```diff
@@ -344,15 +344,15 @@ func (p *standardDynamicScaleDownProcessor) isScaleUpNeededAfterCandidateCluster
 			dryRun:                                true,
 		}
 
-		glog.Infof("evaluating whether deleting the cluster with cluster_id %s would trigger scale up for locator '%+v'", p.clusterID, currLocator)
+		glog.Infof("evaluating whether deleting the cluster with cluster id '%s' would trigger scale up for locator '%+v'", p.clusterID, currLocator)
 		shouldScaleUp, err := dynamicScaleUpProcessor.ShouldScaleUp()
 		if err != nil {
-			glog.Infof("scale up evaluation results returned an error. Not removing cluster with cluster_id %s:", suCount.ClusterId)
+			glog.Infof("scale up evaluation results returned an error. Not removing cluster with cluster id '%s':", suCount.ClusterId)
```
Comment (JameelB) [inline internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go:350]:
```suggestion
			glog.Infof("scale up evaluation results returned an error. Not removing cluster with cluster id '%s'", suCount.ClusterId)
```

**Review by miguelsorianod — APPROVED** [review]

**Comment by miguelsorianod** [comment]
As a side comment, the addition of quotes to a value in Go when using Printf can be done by writing `'%s` or equivalently using `%q`

**Comment by machi1990** [comment]
> As a side comment, the addition of quotes to a value in Go when using Printf can be done by writing `'%s'` or equivalently using `%q`

Thanks. TIL. I'll go with this one.

File: internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go [hunk 3]
```diff
@@ -90,7 +90,7 @@ func (m *DynamicScaleDownManager) processDynamicScaleDownReconcileEvent() error
 		clusterID := suCount.ClusterId
 		existing := processedClusters[clusterID]
 		if existing.processed { // skip already processed
-			glog.V(10).Infof("cluster with cluster_id %s is already processed", suCount.ClusterId)
+			glog.V(10).Infof("cluster with cluster id '%q and status %q is already processed. Skipping it from scale down evaluation", suCount.ClusterId, suCount.Status)
```
Comment (JameelB) [inline internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go:93]: Extra `'` with `%q`. I see it in multiple places in this file

```suggestion
			glog.V(10).Infof("cluster with cluster id %q and status %q is already processed. Skipping it from scale down evaluation", suCount.ClusterId, suCount.Status)
```

**Pull request merged**